### PR TITLE
Dark mode has been added to the Dashboard and other pages.

### PR DESCRIPTION
**Pull request by Nikhil1017**
## Description 📝
Dark mode with appropriate colors has been added as per instructions.

## Screenshots

image
image

### CODE REVIEW

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy I have added it.

File: Components/RightSideBar/RightSideBar.tsx [hunk 1]
```diff
@@ -2,30 +2,34 @@ import React from "react";
 import Image from "next/image";
 import Calendar from "./Calendar";
 import Notes from "./Notes";
+import { DarkModeToggle } from "../components";
```
Comment (lalitmee) [inline Components/RightSideBar/RightSideBar.tsx:5]: Can't we import it like this?
```suggestion
import DarkModeToggle from "../components/DarkModeToggle";
```

File: Components/LogoWithName.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import Image from "next/image";
 const LogoWithName = ({ width, height }: any) => {
   return (
-    <div className="flex  items-center   w-full h-fit  justify-center ">
-      <Image src="./logo.svg" width={width} height={height} alt="logo" />
+    <div className="flex  items-center   w-full h-fit  justify-center">
+     <Image width={40} height={40} src="./logo.svg" alt="logo" className="dark:hidden"/>
+          <Image width={40} height={40} src="./logodark.svg" alt="logo" className="hidden dark:block"/>
```
Comment (lalitmee) [inline Components/LogoWithName.tsx:6]: How are these two images working?

**Review by lalitmee — CHANGES_REQUESTED** [review]
Do we need to add an extra class for dark mode at every place? 

I think we should be able to modify the `tailwind` theme object for `dark` and `light` modes and then it should take the correct colors for whatever mode by itself.

File: Components/RightSideBar/RightSideBar.tsx (hunk 1 above)
Comment (Nikhil1017) [reply Components/RightSideBar/RightSideBar.tsx:5]: Hi @lalitmee thanks for your comment, when I tried that, this is what I got 
image
But based on your suggestion I changed it to like this and it works
image

File: Components/LogoWithName.tsx (hunk 2 above)
Comment (Nikhil1017) [reply Components/LogoWithName.tsx:6]: Unfortunately the fill aspect does not work when we use Image component in tailwind, So we have to use 2 seperate Images, 1 for dark mode and 1 for light mode. When we use dark mode, dark:hidden class hides the light mode logo and when we use light mode, the dark mode logo is hidden by the hidden attribute. You can refer tailwind docs for more info.

File: Components/RightSideBar/RightSideBar.tsx (hunk 1 above)
Comment (Nikhil1017) [reply Components/RightSideBar/RightSideBar.tsx:5]: > Do we need to add an extra class for dark mode at every place?
> 
> I think we should be able to modify the `tailwind` theme object for `dark` and `light` modes and then it should take the correct colors for whatever mode by itself.
 I am not aware of such feature. But I think in doing so we would lose scope for customization for dark mode. I think this also makes it more easy to read and understand.

**Pull request closed without merging**